suite/tests/CMakeLists.txt

```diff
@@ -2460,12 +2460,17 @@ endif ()
       # Both versions utilize the dynamorio_static_nohide as a base because
```

Comment:
> xref i#2204 for having ctest actually re-run these tests.

nit: s/x/X/

> Issue: #3538

s/38/38, #2204/ for a real link.

Reply:
Done.